src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -266,12 +274,14 @@ public void assertContains(JsonElement expectedJson) {
      * @param expectedMessage - the expected response json array
      */
     public void assertContains(String expectedMessage) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (message != null) {
-            success = message.contains(expectedMessage) ? Success.PASS : Success.FAIL;
+            pass = message.contains(expectedMessage) ? true : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)